### ISSUE TITLE
AEF，LED

### ISSUE THREAD

**Issue by BigdateAB** [Jumponthemoon/WeatherGS#2]
I couldn't find the code for the AEF and LED modules mentioned in the paper. Also, what changes did you make to the original 3DGS code? Looking forward to your reply.

**Comment by Jumponthemoon** [comment]
Hi, 

Thank for the interest. We've just released the code for AEF and LED.

Regarding the original 3DGS code: the core masking logic can be found in train.py from line 87 to line 93. 
```python
gt_image = viewpoint_cam.original_image.cuda()
mask = viewpoint_cam.is_masked
if mask is not None:
    mask = mask.cuda()
    gt_image[mask] = image.detach()[mask]
Ll1 = l1_loss(image, gt_image)
loss = (1.0 - opt.lambda_dssim) * Ll1 + opt.lambda_dssim * (1.0 - ssim(image, gt_image))

```
Occluded regions are masked to ensure loss is only computed on visible pixels.
